lib.php

```diff
@@ -99,7 +99,7 @@ function zoom_add_instance(stdClass $zoom, ?mod_zoom_mod_form $mform = null) {
         $zoom->breakoutrooms = $breakoutrooms['zoom'];
     }
 
-    $response = zoom_webservice()->create_meeting($zoom);
+    $response = zoom_webservice()->create_meeting($zoom, $mform->_cm->id);
```

Comment:
Assumes $mform will not be null. Would it be safe to null coalesce to `$zoom->coursemodule`? The core module mod_assign seems to use that value.

Reply:
If this is safe (security wise) and reliable (see previous comment):
```suggestion
    $response = zoom_webservice()->create_meeting($zoom, $zoom->coursemodule);
```